3party/pugixml/CMakeLists.txt

```diff
@@ -8,3 +8,6 @@ set(SRC
 )
 
 add_library(${PROJECT_NAME} ${SRC})
+
+target_include_directories(${PROJECT_NAME} PUBLIC pugixml/src)
+add_library(pugixml::pugixml ALIAS pugixml)
```

Comment:
Is this alias really necessary now?

Reply:
Yes it is. The target is different when used from 3party vs the system dep so this is to prevent conditional statements everywhere we link against it.

Reply:
I meant that you didn't remove pugixml::pugixml in other CMakeLists.txt files.

Reply:
I'm not sure I understand. The system library provides `pugixml::pugixml`, the 3party version provides `pugixml` instead. To make it work everywhere with both system and 3party version this alias is used.

Reply:
Either you or I am misunderstanding something. I can't remove those changes as that won't work with the system-provided pugixml. The system version doesn't provide `pugixml` as a target, it provides `pugixml::pugixml`.

Reply:
I removed this line. Also, I removed other mentioned lines in CMakeLists.txt files. And it works on Mac with and without system pugixml installed. 

Doesn't it work for you? What kind of error do you get?

Reply:
Honestly, I believe you. :shrug: It didn't work for me at the time but this PR is so long open now that I don't really remember the details. I removed the alias, should be fine now.